rosdoc2/verbs/build/impl.py

```diff
@@ -102,21 +103,20 @@ def prepare_arguments(parser):
 
 
 def main(options):
+    """Execute the program, catching errors."""
     try:
         return main_impl(options)
-    except Exception as e:
+    except Exception as e:  # noqa: B902
```

Comment:
@clalancette hmm, `B902`? The documentation I find suggests this is for methods (?).

Reply:
As far as I can tell, `flake8-blind-except` throws a "B902 blind Exception: statement" error on a blind except like this one.  So this disables the warning just on this line.

Reply:
Ha, [`flake8-bugbear`](https://pypi.org/project/flake8-bugbear/) overloads B902...